docker/kernel/linuxkit/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM linuxkit/kernel:4.9.184 AS ksrc
+FROM alpine:3.4 AS probe-build
+ARG FALCOVER=0.17.0
```

Comment:
Why not simply use a "long name" for those variables? 🙂 
```
ARG FALCO_VERSION=0.17.0
ARG SYSDIG_VERSION=0.26.2
```

Reply:
+1